PF/Assignment1/Node.java

```diff
@@ -0,0 +1,18 @@
+package prg3;
+
+//node class
+public class Node<T> {
+	public T nodeValue;
+	public Node<T> next;
+
+	Node() {
```

Comment:
ACTION: No need to create default constructor here as initially nodeValue and next will be null.